linker/shared/src/main/scala/org/scalajs/linker/analyzer/Analyzer.scala

```diff
@@ -1478,6 +1479,10 @@ private class AnalyzerRun(config: CommonPhaseConfig, initial: Boolean,
         _classSuperClassUsed.set(true)
       }
     }
+
+    for (prop <- data.invalidLinkTimeProperties) {
+      _errors ::= InvalidLinkTimeProperty(prop, from)
+    }
```

Comment:
This particular method is very performance-sensitive. We should guard the creation of a lambda to only the cases where the list is non-empty. It will be very rare that it is indeed non-empty, so this is worth it:
```suggestion
    if (data.invalidLinkTimeProperties.nonEmpty) {
      for (prop <- data.invalidLinkTimeProperties) {
        _errors ::= InvalidLinkTimeProperty(prop, from)
      }
    }
```